holidays/countries/saint_vincent_and_the_grenadines.py

```diff
@@ -0,0 +1,100 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.observed_holiday_base import (
+    SAT_SUN_TO_NEXT_MON,
+    ObservedHolidayBase,
+)
+
+
+class SaintVincentAndTheGrenadines(ObservedHolidayBase, ChristianHolidays, InternationalHolidays):
+    """Saint Vincent and the Grenadines holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Saint_Vincent_and_the_Grenadines>
+        * <https://www.timeanddate.com/holidays/saint-vincent-and-the-grenadines/>
+        * <https://www.timeanddate.com/holidays/saint-vincent-and-the-grenadines/national-spiritual-baptist-day>
+        * <https://web.archive.org/web/20250214232128/https://pmoffice.gov.vc/pmoffice/index.php/public-holidays>
+        * <https://web.archive.org/web/20250607111253/http://web.archive.org/screenshot/https://www.stvincenttimes.com/august-3rd-and-4th-2020-declared-public-holidays-in-svg/>
+    """
+
+    country = "VC"
+    default_language = "en_VC"
+    observed_label = tr("%s (observed)")
+    supported_languages = ("en_US", "en_VC")
+    start_year = 1979
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day(tr("New Year's Day")))
```

Comment:
```suggestion
        self._add_new_years_day(tr("New Year's Day"))
```
I have not found any cases of moving this holiday.

Reply:
@KJhellico https://www.timeanddate.com/holidays/saint-vincent-and-the-grenadines/2011

Reply:
Timeanddate is not a reliable enough source (for 2022, it also shows January 3 as day off). Or perhaps it was a one-time additional holiday.

Reply:
@KJhellico but it was mentioned as a source in[ issue's ](https://github.com/vacanza/holidays/issues/1263)description that's why I used it